### PR TITLE
[Task list] Show week task list

### PR DESCRIPTION
**Pull request by piton13**
This pull request should be merged **only** after merging "show today task list" - pull request.

### CODE REVIEW

File: client/js/app/main/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ var mainModuleDependencies = []
     .concat(require('./modules'));
 
 var mainModule = angular.module('mainModule', mainModuleDependencies)
-    .config(require('./main.config'));
+    .config(require('./main.config'))
+    .value('nowValue', new Date());
```
Comment (kucherenko) [inline client/js/app/main/index.js:9]: remove this value, we always have new Data() and don't need value for it

File: client/js/app/main/modules/menu/menu.directive.controller.js [hunk 2]
```diff
@@ -4,15 +4,38 @@ var angular = require('angular'),
...
             isVisible = !vm.isVisible;
+        }
 
         vm.isVisible = isVisible;
         event.stopPropagation();
     });
-};
\ No newline at end of file
+
+    todayService.getTasks(function (res) {
```
Comment (kucherenko) [inline client/js/app/main/modules/menu/menu.directive.controller.js:23]: Refactor this method, make all steps more clear. Move code to separate methods. For now this method is unreadable.

File: client/js/app/main/modules/week/week.config.js [hunk 3]
```diff
@@ -4,9 +4,11 @@
 module.exports = function ($stateProvider) {
     $stateProvider.state('main.week', {
         url: '/week',
+        controller: require('./week.controller'),
+        controllerAs: 'week',
         templateUrl: {
```
Comment (kucherenko) [inline client/js/app/main/modules/week/week.config.js:9]: Make $templateCache object and don't use root: __dirname for template. This issue connected to all templates.

For resolve it we need to create gulp tusk for it.

File: client/js/app/main/modules/week/week.controller.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+ * @ngInject
+ */
+module.exports = function (todayService, $filter, nowValue) {
+    var vm = this,
+        oneDay = 86400000,
+        parsedDate = Date.parse(nowValue);
+    this.tasks = [];
+    this.todayLong = $filter('date')(nowValue, "d MMM yyyy").toLowerCase();
+
+    this.weekDays = [
```
Comment (kucherenko) [inline client/js/app/main/modules/week/week.controller.js:11]: Move logic of date manipulation from controller to service, if we have no service for it - create it.

File: client/js/app/main/modules/week/week.controller.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+        date:parsedDate + 6*oneDay,
+        dateFiltered: $filter('date')(parsedDate + 6*oneDay, "d MMM yyyy").toLowerCase()
+      }
+    ];
+
+    todayService.getTasks(function (res) {
+        vm.tasks = res.data;
+    });
+
+    console.log(oneDay);
```
Comment (kucherenko) [inline client/js/app/main/modules/week/week.controller.js:46]: remove debug info and comments

File: client/js/app/main/modules/week/week.config.js (hunk 3 above)
Comment (borys-ihnatyev) [inline client/js/app/main/modules/week/week.config.js:9]: It's my fault, I'll refactor it furture.

Comment (borys-ihnatyev) [inline client/js/app/main/modules/week/week.config.js:9]: Done a23f3517

**Pull request closed without merging**